apps/studio/components/interfaces/Organization/BillingSettings/Subscription/SubscriptionPlanUpdateDialog.tsx

```diff
@@ -43,7 +43,7 @@ const getRandomTweet = () => {
 
 const PLAN_HEADINGS = {
   tier_pro:
-    'the Pro plan and create unlimited projects, daily backups and premium support when you need it',
+    'the Pro plan to unlock unlimited projects, daily backups, and priority support whenever you need it',
```

Comment:
We mention priority support for the team plan already - would rename this slightly maybe even just leave it the word

```suggestion
    'the Pro plan to unlock unlimited projects, daily backups, and support whenever you need it',
```

Reply:
Or perhaps "email support"

Reply:
I llike "email support", I think it sets the right expectations.